File: LpSolve/Result/UnboundedSeidelResult.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using LpSolve.Elements;

namespace LpSolve.Result
{
	public class UnboundedSeidelResult : SeidelResult
	{
		public override SeidelResult Resolve(HalfSpace halfSpace)
		{
			throw new NotImplementedException();
		}
	}
}

File: cpp/Result/UnboundedSeidelResult.cpp
#include "UnboundedSeidelResult.h"
#include "../Elements/HalfSpace.h"
#include "../exceptionhelper.h"

using namespace LpSolveCpp::Elements;

namespace LpSolveCpp
{
	namespace Result
	{

		SeidelResult *UnboundedSeidelResult::Resolve(HalfSpace *halfSpace)
		{
			throw NotImplementedException();
		}
	}
}
